ui-manchette-with-spacetimechart/src/index.ts

```diff
@@ -0,0 +1,4 @@
+import '@osrd-project/ui-core/dist/theme.css';
+import './styles/main.css';
```

Comment:
Since it is used only in the story, can we import it in the story ?
The index file is what is exposed outside the library, I think it's not a good practice to expose things that the users won't use

Reply:
As we used postcss it's not working before the build in dist theme.css

Reply:
Nice !